solved_exercises/dani/quadratic.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from cmath import sqrt
+
+
+def polynome(a, b, c):
+    """This function calculates values of a univariate quadratic equation
+    x1 = -b + sqrt(b^2 - 4ac)
+    x2 = -b - sqrt(b^2 - 4ac)
+    """
+    if not all(isinstance(i, (int, complex, float)) for i in [a, b, c]):
+        raise TypeError("Use numbers as input")
+
+    if a == 0:
```

Comment:
You don't need to raise this exception. Python will do it for you anyway and it has a clearer message than this one.